### PR TITLE
lokomotive: Add entry for 0.6

### PR DESCRIPTION
**Pull request by surajssd**
### Changes

- This commit adds an entry for `0.6`.
- Point the `0.5` to `v0.5.0`.
- Fix the indentation of YAML.

### Testing done

None, I am not sure how test locally.

**NOTE:** Will be merged after the release is done for lokomotive https://github.com/kinvolk/lokomotive/pull/1292

### CODE REVIEW

**Comment by surajssd** [comment]
@iaguis is this the right way to do it?

**Comment by joaquimrocha** [comment]
Yes. If you want to keep the version 5 docs available this is the way to do that.
The `latest` version alias will be pointing to the first entry in that list, so it's also good.

File: docs/lokomotive/_index.md [hunk 1]
```diff
@@ -3,10 +3,14 @@ content_type: lokomotive
...
-    branch: "master"
-    dir: "docs"
+- repo: https://github.com/kinvolk/lokomotive.git
+  name: "0.6"
+  branch: "master"
+  dir: "docs"
+- repo: https://github.com/kinvolk/lokomotive.git
+  name: "0.5"
+  branch: "v0.5.0"
+  dir: "docs"
```
Comment (iaguis) [inline docs/lokomotive/_index.md:13]: I think this should work.

```suggestion
- repo: https://github.com/kinvolk/lokomotive.git
  name: "0.6"
  branch: "v0.6.0"
  dir: "docs"
- repo: https://github.com/kinvolk/lokomotive.git
  name: "current"
  branch: "master"
  dir: "docs"
```

**Comment by iaguis** [comment]
> Yes. If you want to keep the version 5 docs available this is the way to do that.

v0.5.0 doesn't have docs, We want to have v0.6.0 and then current master, I wrote a suggestion.

File: docs/lokomotive/_index.md (hunk 1 above)
Comment (surajssd) [reply docs/lokomotive/_index.md:13]: Wouldn't removing `0.5.0` tag unlink all the URLs with `0.5.0` in them? Or there is a provision to redirect such URLs?

Comment (iaguis) [reply docs/lokomotive/_index.md:13]: That's a good point. I think we can keep `0.5` and point it to the first commit with the new docs (which is not going to be `v0.5.0` but is as closest as possible) to keep the current `0.5` links.

Comment (iaguis) [reply docs/lokomotive/_index.md:13]: Filed https://github.com/kinvolk/docs/pull/11

**Comment by iaguis** [comment]
This was superseded by #11.

**Pull request closed without merging**